lib/src/replaced_element.dart

```diff
@@ -71,7 +71,12 @@ class ImageContentElement extends ReplacedElement {
     this.src,
     this.alt,
     dom.Element node,
-  }) : super(name: name, style: style, node: node);
+  }) : super(
```

Comment:
I don't spend a lot of time doing Dart, but has this been formatted? I understand that the `super` is too long for the line, but it looks strange to me. If this is how it is supposed to be, I don't mind, just looks odd to my Go developer eyes

Reply:
Yeah dartfmt produces consisted yet sometimes odd looking code.